### PR TITLE
updating sdk and extension package references

### PR DESCRIPTION
**Pull request by brettsam**

### CODE REVIEW

File: src/WebJobs.Script.Host/WebJobs.Script.Host.csproj [hunk 1]
```diff
@@ -226,8 +226,10 @@
...
     <Error Condition="!Exists('..\..\packages\StyleCop.MSBuild.4.7.50.0\build\StyleCop.MSBuild.Targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\StyleCop.MSBuild.4.7.50.0\build\StyleCop.MSBuild.Targets'))" />
     <Error Condition="!Exists('..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets'))" />
+    <Error Condition="!Exists('..\..\packages\Microsoft.Azure.DocumentDB.1.6.3\build\Microsoft.Azure.DocumentDB.targets')" Text="$([System.String]::Format('$(ErrorText)', '..\..\packages\Microsoft.Azure.DocumentDB.1.6.3\build\Microsoft.Azure.DocumentDB.targets'))" />
   </Target>
   <Import Project="..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets" Condition="Exists('..\..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" />
+  <Import Project="..\..\packages\Microsoft.Azure.DocumentDB.1.6.3\build\Microsoft.Azure.DocumentDB.targets" Condition="Exists('..\..\packages\Microsoft.Azure.DocumentDB.1.6.3\build\Microsoft.Azure.DocumentDB.targets')" />
```
Comment (mathewc) [inline src/WebJobs.Script.Host/WebJobs.Script.Host.csproj]: What is this new build package?

**Comment by mathewc** [comment]
Looks good to me - just wondering what that new build package is. If it is required, then just check it in.

File: src/WebJobs.Script.Host/WebJobs.Script.Host.csproj (hunk 1 above)
Comment (brettsam) [inline src/WebJobs.Script.Host/WebJobs.Script.Host.csproj]: Docdb adds it -- this is all the result of a Nuget update. I didn't do anything fancy.

**Comment by mathewc** [comment]
:ship:

**Pull request merged**